src/redux/trainers/reducer.js

```diff
@@ -3,7 +3,8 @@ import * as trainersConstant from './constants';
 const initialState = {
   trainers: [],
   isLoading: true,
-  error: ''
+  error: '',
```

Comment:
In the index you aren't using the error and the success, if you want to handle the errors with this you should apply them instead of the modal